packages/swingset-theme-hashicorp/src/components/side-nav/toggle-button.tsx

```diff
@@ -0,0 +1,22 @@
+import { IconChevronLeft24 } from '@hashicorp/flight-icons/svg-react/chevron-left-24'
+import { cx } from 'class-variance-authority'
+
+type ToggleButtonProps = {
+  toggle: () => void
+  isOpen: boolean
+}
+
+export function ToggleButton({ toggle, isOpen }: ToggleButtonProps) {
+  return (
+    <button
+      aria-label="Toggle the navigation menu between open and closed states"
```

Comment:
Ideally this label will be something that a user taking advantage of voice control will be able to target, based on the element's context. I'm thinking we could do something like this:

```
isOpen: aria-label="Collapse navigation menu"
!isOpen: aria-label="Expand navigation menu"
```